src/media/js/views/homepage.js

```diff
@@ -0,0 +1,67 @@
+define('views/homepage',
+    ['apps', 'core/capabilities', 'core/l10n', 'core/models', 'core/z',
+     'templates', 'spatial-navigation'],
+    function(apps, caps, l10n, models, z,
+             nunjucks, SpatialNavigation) {
+    var gettext = l10n.gettext;
+    var appsModel = models('apps');
+
+    // Initialize spatial navigation.
+    SpatialNavigation.init();
+
+    // Define the navigable elements.
+    SpatialNavigation.add({
+        selector: '.focusable'
+    });
+
+    z.page.on('loaded', function() {
+        // Add 'tabindex="-1"' to "currently-existing" navigable elements.
+        SpatialNavigation.makeFocusable();
+
+        // Focus the first navigable element.
+        SpatialNavigation.focus();
+    });
+
+    z.page.on('focus', '.focusable', function() {
+        // Update app preview area with current focused app.
+        z.page.find('.app-preview').html(
+            nunjucks.env.render('_includes/app_preview.html', {
+                app: appsModel.lookup($(this).data('slug'))
+            })
+        );
+    });
+
+    z.page.on('sn:enter-down', '.focusable', function() {
```

Comment:
We probably want `sn:enter-up` but it's tough to say on a TV remote. This will trigger the install on 'enter' but it will keep the event on holding the key down. Should probably be tested on a remote if possible.